packages/database/src/darkchat/index.ts

```diff
@@ -111,11 +111,7 @@ export class _DarkchatDB {
     return { id: result[0].id, label: result[0].label };
   }
 
-  async joinChannel(
-    channelIdentifier: string,
-    userIdentifier: string,
-    isOwner: boolean,
-  ): Promise<number> {
+  async joinChannel(channelIdentifier: string, userIdentifier: string, isOwner: boolean): Promise<number> {
```

Comment:
This shouldn't be collapsed, eslint show throw warnings/errors about this

Reply:
also no warnings or errors here as far as im aware. this is the first im ever hearing about this? was this a previous version because I upgraded eslint in the packages at some point im pretty sure

Reply:
looking at much of the db class exports theyre all written in a similar fashion. is such a change necessary? if thats the case then shouldn't all of the classes be modified as well?